src/components/Alert.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './Alert.css';
+import CloseIcon from '@mui/icons-material/Close';
+
+export interface AlertProps {
+  id: string;
+  type: string;
+  title: string;
+  description: string;
+  onClose?: () => void;
+}
+
+export const Alert = ({id, type, title, description, onClose}: AlertProps) => {
```

Comment:
breaking this out to a second line would help readability and match the recommended pattern (including the return type:

export const Alert = (props: AlertProps): React.ReactNode => {
  const {id, type, title, description, onClose } = props;
  //...